src/QueryEditor.tsx

```diff
@@ -41,24 +42,25 @@ export const QueryEditor: React.FC<Props> = props => {
       props.onRunQuery();
     }
   }, []);
```

Comment:
I guessed that we only want to check this once on component mount?

Reply:
yes 👍